cli/tests/install.bats

```diff
@@ -375,45 +381,32 @@ EOF
     run "$FLOX_BIN" install nodejs@14.16.1
 
   assert_failure
-  assert_output "$(cat <<EOF
+  assert_output "$(
+    cat << EOF
 ❌ ERROR: resolution failed: constraints for group 'toplevel' are too tight
 
    Use 'flox edit' to adjust version constraints in the [install] section,
    or isolate dependencies in a new group with '<pkg>.pkg-group = "newgroup"'
 EOF
-)"
+  )"
 }
 
 # ---------------------------------------------------------------------------- #
 
 @test "flake: github ref added to manifest" {
-  skip "installables will be fixed in a follow-up PR"
-
   "$FLOX_BIN" init
-  input_flake="github:foo/bar"
+  input_flake="github:nixos/nixpkgs/$PKGDB_NIXPKGS_REV_OLD#hello"
   run "$FLOX_BIN" install "$input_flake"
   assert_success
-  installed_flake=$(tomlq -r -c -t ".install.bar" "$MANIFEST_PATH")
+  installed_flake=$(tomlq -r -c -t ".install.hello" "$MANIFEST_PATH")
   assert_equal "$installed_flake" "flake = \"$input_flake\""
 }
 
 @test "flake: https ref added to manifest" {
-  skip "installables will be fixed in a follow-up PR"
-
-  "$FLOX_BIN" init
-  input_flake="https://github.com/foo/bar/archive/main.tar.gz"
-  run "$FLOX_BIN" install "$input_flake"
-  assert_success
-  installed_flake=$(tomlq -r -c -t ".install.bar" "$MANIFEST_PATH")
-  assert_equal "$installed_flake" "flake = \"$input_flake\""
-}
-
-@test "flake: fallback id added to manifest" {
```

Comment:
This was removed because we no longer use a fallback, we always parse something from the URL